scielomanager/editorialmanager/tests/test_forms.py

```diff
@@ -180,6 +180,9 @@ def test_ADD_board_member_valid_POST_is_valid(self):
         """
         User of the group "Editors" successfully ADD a new board member
         """
+
+        from django_countries import countries
```

Comment:
Algumas justificativas para deixar o import no top level
- indicado pela **PEP8**: ``Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.``
- é o que acostumamos fazer.
- em quanto a eficiência, deixar o import no top level, é rápido. E deixar o import dentro de uma classe ou função, deixa a execução dela mais demorada.[1]

[1] http://stackoverflow.com/a/128577/1503